hera_qm/ant_metrics.py

```diff
@@ -0,0 +1,394 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import aipy as a 
+from copy import deepcopy
+from pyuvdata import UVData
+import json
+
+
+#######################################################################
+#Low level functionality that is potentially reusable
+#######################################################################
+
+
+def per_antenna_modified_z_scores(metric):
+    '''For a given metric, stored as a (ant,antpol) dictonary, computes the per-pol modified z-score 
+    for each antenna, which is the metrics, minus the median, divided by the median absolute deviation.'''
+    zscores = {}    
+    antpols = set([key[1] for key in metric.keys()])
+    for antpol in antpols:            
+        values = np.array([val for key,val in metric.items() if key[1]==antpol])
+        median = np.nanmedian(values)
+        medAbsDev = np.nanmedian(np.abs(values - median))
+        for key,val in metric.items(): 
+            if key[1]==antpol:
+                zscores[key] = 0.6745*(val - median) / medAbsDev 
```

Comment:
I've assigned myself an issue: #27 

This should wait until we're comfortable with the operation and effectiveness of the metrics.